src/main/assets/scss/home.scss

```diff
@@ -0,0 +1,4105 @@
+// Home page css override
```

Comment:
can we not just import this? thats if we actually need it

Reply:
If anyone else reads this, we've decided to include a heavily pruned version of the CSS from the initial commit (4k lines down to ~180ish) as there were issues with importing the relevant gov.uk CSS file.